dev/dune-workspace.all

```diff
@@ -0,0 +1,14 @@
+(lang dune 1.2)
+
+; Add custom flags here. Default developer profile is `dev`
```

Comment:
Maybe remove this comment that was copy-pasted.

Reply:
You are correct in that we should instead define the flags in common file and use `(include ...)`, added to the todo list.